package/kedro_viz/models/graph.py

```diff
@@ -43,10 +43,6 @@ def _strip_namespace(name: str) -> str:
     return re.sub(pattern, "", name)
 
 
-def _strip_tags(name: str) -> str:
-    return name.replace("<", "&lt;").replace(">", "&gt;")

```

Comment:
I think the solve for this is to leave the `<>` in place in the backend and then change it to use the HTML entities _only where we need it_ in the frontend.